plugin.rb

```diff
@@ -8,38 +8,39 @@
 
 courses = Hash.new
 courses = {
+    11 => "e11_scratch_00",
+    57 => "e12_scratch_00",
+    85 => "e13_text_prog_00",
     36 => "e13_real_prog_00",
     37 => "e14_minecraft_00",
+    90 => "e20_prog_fundamentals_00",
     45 => "e21_prog_concepts_00",
     31 => "e22_wd1_00",
-    58 => "e23",
-    46 => "e24",
+    58 => "e23_wd2_00",
+    46 => "e24_python_game_dev_00",
     13 => "m112_intro_prog_py_00",
-    14 => "m12_html_css_00",
+    84 => "m12_python_beyond_basics_00"
+    14 => "m13_html_css_00",
     15 => "m13_js_00",
     16 => "M14_vr_00",
+    89 => "m15_web_dev_00",
```

Comment:
This will be m21_something_something_00. Let me ask the new Curriculum Developer what the URL structure is and we can update.

Reply:
Here's the URL: m21_resp_web_dev_00